common/views/components/Buttons/Control/Control.js

```diff
@@ -22,7 +22,7 @@ type Props = {|
   disabled?: boolean,
   ariaControls?: string,
   ariaExpanded?: boolean,
-  clickHandler?: (event: Event) => void,
+  clickHandler?: (event: Event) => any,
```

Comment:
Any reason for this?

Reply:
Needed to use a function tagged with `async`, which returns a `Promise`, so `void` was breaking Flow. Could be `void | Promise` ?

Reply:
`void | Promise<void>` makes sense.